YALContextMenu/YALContextMenuTableView.m

```diff
@@ -94,6 +94,7 @@ - (void)showInView:(UIView *)superview withEdgeInsets:(UIEdgeInsets)edgeInsets a
         } completion:^(BOOL finished) {
             [self show:YES visibleCellsAnimated:YES];
             [self setUserInteractionEnabled:YES];
+            self.contentOffset = contentOffset;
```

Comment:
@pventura1976 thank you for PR.
Content offset will be applied only once until user scroll a table view. In your example If user scrolls table view from its initial position in the bottom direction than table view returns to content offset {0,0} and it looks strange.
Currently this PR is in git conflict mode, and I am going to close it.
If you need this changes be merged in provide a new PR with description why you need this functionality and  how you are going to use it. Thanks.